src/parsers/parseArgs.ts

```diff
@@ -42,8 +44,12 @@ export function parseArgs(str: string) {
     args.push(currentArg);
     currentArg = "";
   }
+  function substitute(char: string) {
+    if (!insideSingleQuotes && !insideDoubleQuotes) currentArg += char;
+  }
 
-  loop: for (const char of str) {
+  loop: for (let i = 0; i < str.length; i++) {
```

Comment:
why the conversion to a cstyle for loop?

Reply:
An oversight, I thought I was going to se the index.